README.md

```diff
@@ -77,33 +77,25 @@ To run linters, code formatters, and test suites all together, use `test`:
 
 ### Code quality checks
 
-PySTAC uses [flake8](http://flake8.pycqa.org/en/latest/) and [yapf](https://github.com/google/yapf) for code formatting and style checks.
-
-To run the flake8 style checks:
+PySTAC uses [flake8](http://flake8.pycqa.org/en/latest/) and [Black](https://github.com/psf/black) for code formatting and style checks. To run both:
```

Comment:
Not sure how others feel about this, but it seems like this section is sort of redundant with the [Contributing docs](https://github.com/stac-utils/pystac/blob/8946c9b192e20f53b75196915bb078c91954ae76/docs/contributing.rst) and maybe should be removed.

I'm fine if we want to make that change separately since it was not the goal of this PR, but if that's the case, we may want to mention `mypy` and `doc8` here as well so it's at least consistent with those Contributing docs for now.

Reply:
+1 on removing and deferring to Contributing

Reply:
Opened #440 to track consolidating these docs. @l0b0 Feel free to tackle that as part of this PR if you want, but I'm also fine with just updating the README to be accurate for now and updating those docs in a separate PR.

Reply:
Cheers; I've removed the section in this PR.